snowplow/iglu-client-embedded/schemas/com.metabase/embedded_analytics_js/jsonschema/1-0-0

```diff
@@ -0,0 +1,199 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Embedded Analytics JS configuration",
+  "self": {
+    "vendor": "com.metabase",
+    "name": "embedded_analytics_js",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "event": {
+      "description": "The type of event being recorded. For example, then the Embedded Analytics JS is initialized, or when the Metabase configuration is updated.",
+      "type": "string",
+      "enum": [
+        "setup"
+      ],
+      "maxLength": 1024
```

Comment:
(I guess this is always "setup" right? I guess we set maxLength=1024 for backwards compat when we need more enums.)

Reply:
I see we always set `maxLength` 1024 on all string on snowplow schema. So I followed that pattern. Right now it could be only `setup` we could track when `window.metabaseConfig` changes if we want in the future. Maybe there would be new events too who knows.